molecule/system_settings_logging/molecule.yml

```
@@ -0,0 +1,67 @@
+---
+scenario:
+    name: system_settings_general
```

I would change that to `system_settings_logging` to avoid confusion:
```suggestion
    name: system_settings_logging
```